docs/services/navigation/_index.md

```diff
@@ -13,12 +13,15 @@ tags: ["navigation", "services", "base", "rover"]
 
 The Navigation service is the stateful definition of Viam's [Motion Service](/services/motion/).
 It uses GPS to autonomously navigate a rover [base](/components/base/) to user defined endpoints called `Waypoints`.
-Once these waypoints are added and the mode of the service is [set to `waypoints`](#setmode), the service begins to define the robot's path.
+Once these waypoints are added and the mode of the service is [set to `MODE_WAYPOINT`](#setmode), the service begins to define the robot's path.
 
 ## Configuration
 
 You must configure a [base](/components/base/) with a [movement sensor](/components/movement-sensor/) as part of your robot to configure a Navigation service.
 
+The movement sensor must support `GetPosition`, `GetCompassHeading`, or `Orientation` in its {{< glossary_tooltip term_id="model" text="model's" >}} implementation of the [Sensor API](/components/sensor/#api).
```

Comment:
@npentrel hmmm-- this is translated from a direct ask that I'm not sure if I am interpreting correctly: `Add: The movement sensor must support GetPosition and GetCompassHeading or Orientation. If your movement sensor can support both, then the motion planner will default to using the compass heading values`

@tessavitabile Is the following suggestion what your team means by that? If so, I think that would clear up Naomi's question.
And maybe I could provide more info on why this requirement is needed (if it needs combo of two sensor reading types or a GeoPoint)? 

```suggestion
The movement sensor must support both `GetPosition` and `GetCompassHeading`, or just `Orientation`, in its {{< glossary_tooltip term_id="model" text="model's" >}} implementation of the [Sensor API](/components/sensor/#api).
```

Reply:
The movement sensor must support GetPosition, in order to know the robot's current GPS location. It must also support either GetCompassHeading or GetOrientation, in order to know which way the robot is facing. If the movement sensor supports both GetCompassHeading and GetOrientation, then GetCompassHeading will be used.

Reply:
@tessavitabile Perfect, thank you